src/main/java/com/faforever/client/headerbar/UserButtonController.java

```diff
@@ -57,6 +61,10 @@ public void onReport() {
     reportDialogController.show();
   }
 
+  public void onCopyAccessToken() {
+    Platform.runLater(() -> ClipboardUtil.copyToClipboard(tokenRetriever.getAccessToken()));
```

Comment:
The get Access token method should return a mono and not block. And then the fxApplicationExecutor should be used to execute on the platform thread instead of platform.runLater. you can use its as scheduler method and publishOn of the returned mono.

Reply:
Although I am not sure if the clipboard has to be used on the application thread

Reply:
I tried without it and failed without running later.